### DISCUSSION TITLE
Não consigo Importar arquivo para o github

### DISCUSSION THREAD

**Discussion by Tchelovsky** [frontendbr/forum#2106] [Perguntas e Respostas]
Quem puder me ajudar... Sou novo no Git e vi alguns vídeos no youtube de como importar o arquivo para o Git... Clico com o botão direito do mouse faço os passos a passos do Git Bash, porém sempre ocorre o erro ai do print! E olha que já consegui importar dois trabalhos para o Git!

Pode ser que para alguns o processo é bem simples, porem pra mim está dando PUTA de uma dor de cabeça. Quem puder me ajudar

lembrando que os outros projetos fiz exatamente igual porem agora que está dando esse erro! 
![java git](https://user-images.githubusercontent.com/56838742/123339118-654ac200-d520-11eb-819d-820e3a3e35e5.PNG)

**Comment by igorlvicente** [comment]
Tenha certeza que as aspas estão certas em `git commit -m "Exemplo - Exercícios Java"`

Talvez copiaste o comando direto e ele veio com caracteres inválidos.

**Reply by Tchelovsky** [reply]
Vou tentar aqui Igor e volta pra falar se deu certo.
